Project-I-Social-Card/social-card/src/App.css

```
@@ -2,3 +2,8 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.App{
+  width: 60%;
```

A max-width of ~400-600px would make this more readable. 

Generally, you want to limit the width of divs containing text to no wider than 600px because humans find it easier to read columns that wide. 

For instance, adding max width like this:
```suggestion
  width: 60%;
  max-width: 500px;
```

Setting a max width would also negate the need to do most media queries.